chainreg/chainregistry.go

```diff
@@ -377,10 +377,13 @@ func NewChainControl(cfg *Config, blockCache *blockcache.BlockCache) (
 				(cfg.Litecoin.Active && cfg.Litecoin.RegTest) {
 				conn, err := net.Dial("tcp", bitcoindHost)
 				if err != nil || conn == nil {
-					if cfg.Bitcoin.Active && cfg.Bitcoin.RegTest {
+					switch {
+					case cfg.Bitcoin.Active && cfg.Bitcoin.RegTest:
 						rpcPort = 18443
-					} else if cfg.Litecoin.Active && cfg.Litecoin.RegTest {
+					case cfg.Litecoin.Active && cfg.Litecoin.RegTest:
 						rpcPort = 19443
+					case cfg.Bitcoin.Active && cfg.Bitcoin.SigNet:
+						rpcPort = 38332
```

Comment:
@vwoo perhaps something is being morphed before the call is made? We enforce the default port here

Reply:
If I change the signet port from 38332 to 38334 in chainreg/chainparams.go then the default port works.

Just following the pattern in the rest of chainparams.go where the port is always +2.

bitcoind --help | grep rpcport
mainnet: 8332
testnet: 18332
signet: 38332

chainreg/chainparams.go
mainnet: 8334
testnet: 18334
signet: 38332

[Seems like the pattern break is intentional](https://github.com/lightningnetwork/lnd/pull/5025#discussion_r627795943) but maybe there was a mixup between chainregistry.go and chainparams.go?

Reply:
@vwoo see https://github.com/lightningnetwork/lnd/pull/5351.